tests/IntegrationTestDefs.js

```diff
@@ -283,17 +283,21 @@ gpii.tests.dev.testDefs = {
         listener: "gpii.tests.dev.testMultiPrefSetMenu"
     }, { // simulate choosing different pref set
         func: "{that}.app.tray.menu.model.menuTemplate.7.click"
-    }, { // test Active Pref Set changed
-        changeEvent: "{that}.app.tray.menu.applier.modelChanged",
-        path: "menuTemplate",
-        args: [
-            "{that}.app.tray",
-            "{that}.app.tray.menu.model.menuTemplate",
-            /*prefSetItemClickedIdx=*/7,
-            "{that}.app.model.preferences"
-        ],
-        listener: "gpii.tests.dev.testActiveSetChanged"
-    }, {
+    },
+    // Commented after GPII-3826. Need to be changed to meet the new requirements
```

Comment:
I'm wondering what "the new requirements" mean. Are they in a spec? In a JIRA? Can we have it documented and (if makes sense) get them referenced from code?